src/MessageCreationUtil.js

```diff
@@ -214,7 +214,7 @@ export default class MessageCreationUtil {
             streamId,
             requestId,
         }
-        const [messageId, prevMsgRef] = this.createDefaultMsgIdAndPrevRef(destinationAddress.toLowerCase(), publisherId)
+        const [messageId, prevMsgRef] = this.createDefaultMsgIdAndPrevRef(destinationAddress, publisherId)
```

Comment:
Noticed one more cosmetic thing here: the variable name `destinationAddress` is a remnant from the time when the key exchange streamId was just the address. Now that it's `SYSTEM/keyexchange/address`, the variable should be renamed to `keyExchangeStreamId` or similar.

Similar misleading naming is currently happening in other message creation methods here too.

Reply:
Changed `destinationAddress` to `keyExchangeStreamId` in 0b57615.